pkg/kubectl/cmd/create_deamonset.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+)
+
+var (
+	daemonsetLong = templates.LongDesc(i18n.T(`
+	Create a daemonset with the specified name.
+
+	A DaemonSet ensures that all (or some) nodes run a copy of a pod.  As nodes are added to the
+	cluster, pods are added to them.  As nodes are removed from the cluster, those pods are garbage
+	collected.  Deleting a DaemonSet will clean up the pods it created.
+
+	Some typical uses of a DaemonSet are:
+
+	- running a cluster storage daemon, such as 'glusterd', 'ceph', on each node.
+	- running a logs collection daemon on every node, such as 'fluentd' or 'logstash'.
+	- running a node monitoring daemon on every node, such as [Prometheus Node Exporter](
+	  https://github.com/prometheus/node_exporter), 'collectd', Datadog agent, New Relic agent, or Ganglia 'gmond'.
+
+	In a simple case, one DaemonSet, covering all nodes, would be used for each type of daemon.
+	A more complex setup might use multiple DaemonSets for a single type of daemon, but with
+	different flags and/or different memory and cpu requests for different hardware types.
+
+	See more detail information [https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/]
+	`))
+
+	daemonsetExample = templates.Examples(i18n.T(`
+	# Create a new daemonset named my-dea that runs the busybox image.
```

Comment:
More examples is definitely good idea.

Reply:
Do you support node label, maybe?

Reply:
nothing now，we want to make create command simple enough, and then use other commands to edit it.
```
kubectl create ds hello --image=nginx --dry-run -o yaml | kubectl lable --local -f - env=qa -o yaml | kubectl create -f -
```